_data/simple-icons.json

```
@@ -6788,6 +6788,11 @@
             "hex": "EE3524",
             "source": "https://cdn.gurobi.com/wp-content/uploads/2021/02/Gurobi-Optimization_Corporate-Brochure.pdf"
         },
+        {
+            "title": "Gusto",
+            "hex": "F45D48",
+            "source": "https://prod.gusto-assets.com/assets/svg/header/gusto-logo-bfed4fde4bd2b5dd364ab0e58f77fc90aeefa2f1ecdb7fff524279642e23f9c2.svg"
```

```suggestion
            "source": "https://gusto.com"
```
Since the logo is taken from the header, the main web page link would suffice.
The asset link might change any day.